src/app/frontend/replicationcontrollerdetail/replicationcontrollerdetail.html

```diff
@@ -138,22 +140,32 @@ <h1 flex class="md-title kd-replicationcontrollerdetail-app-name">
                   -
                 </span>
               </td>
-              <td class="kd-replicationcontrollerdetail-table-cell"
-                  ng-if="::ctrl.replicationControllerDetail.hasMetrics">
+              <!-- width=1 is a hack to shrink-wrap the column around its content -->
+              <td class="kd-replicationcontrollerdetail-table-cell kd-replicationcontrollerdetail-metric-table-cell"
+                  ng-if="::ctrl.replicationControllerDetail.hasMetrics"
+                  width="1">
                 <span ng-if="::ctrl.hasCpuUsage(pod)">
                   {{::(pod.metrics.cpuUsage | kdCores)}}
                 </span>
-                <span ng-if="::!ctrl.hasCpuUsage(pod)">
-                  -
-                </span>
```

Comment:
Without that when there are no metrics to display columns are empty.

![zrzut ekranu z 2016-02-19 10-01-10](https://cloud.githubusercontent.com/assets/2285385/13170873/b062439c-d6ef-11e5-91e3-77b207e93e48.png)
![zrzut ekranu z 2016-02-19 10-00-10](https://cloud.githubusercontent.com/assets/2285385/13170857/8a18f01e-d6ef-11e5-883a-77192bade3c5.png)

This doesn't look very good. Sometimes we have to wait a bit for metrics to appear so i guess this is needed.